Mixin/Service/DeviceTransfer/DeviceTransferClient.swift

```diff
@@ -223,7 +248,24 @@ extension DeviceTransferClient {
             } catch {
                 Logger.general.error(category: "DeviceTransferClient", message: "Failed to finish command: \(error)")
             }
-            self.stop(reason: .finished)
+            messageProcessor.$progress
+                .receive(on: queue.dispatchQueue)
+                .sink { progress in
+                    if progress == 1 {
```

Comment:
Is it possible for Progress to be inaccurate?

Reply:
Updated with [317d12c](https://github.com/MixinNetwork/ios-app/pull/1293/commits/317d12c20d56d19511cae20ca7db8556bf56b991)